### ISSUE TITLE
Allow users to view segment configuration - read-only

### ISSUE THREAD

**Issue by luk355** [SolidStateGroup/bullet-train-api#2653]
**Is your feature request related to a problem? Please describe.**

Manage segments role is limited only to small subset of users in our setup as misconfigured segment can easily expose features to more users than expected. Other users (mainly QA) still need the ability to view segment configuration so that they can learn what traits needs to be set for the user to belong to that segment.

**Describe the solution you'd like**

Allow users with "View project" permission to access read-only view of segment detail.

**Describe alternatives you've considered**

Add new "View segments" permission.

**Comment by matthewelwell** [comment]
Thanks @luk355, this is interesting. I think perhaps the alternative solution for adding a new 'View segments' permission is probably the better option here as some organisations might have sensitive data contained in their segments which they may not want everyone to see (unlikely but not impossible).

**Comment by luk355** [comment]
Yes, since users without "Manage segments" permissions now cannot view the segment details, adding new "View segments" permission will not change the behavior (permission) for such users.

Any idea if/when will you be able implement this change so that I can update our QA? There is also related request to segment permissions raised few months back: https://github.com/Flagsmith/flagsmith/issues/1676

**Comment by matthewelwell** [comment]
I've added it to our backlog and will attempt to prioritise it, alongside #1676. I cannot give a date just yet as we're pretty flat out, but I would expect we can complete these within the next month or 2.
